electron/legendary/games.ts

```diff
@@ -193,66 +193,68 @@ class LegendaryGame extends Game {
    * @returns The amended install path.
    */
   public async moveInstall(newInstallPath: string) {
-    const {
-      install: { install_path },
-      title
-    } = await this.getGameInfo()
+    const oldInstallPath = (await this.getGameInfo()).install.install_path
 
-    if (isWindows) {
-      newInstallPath += '\\' + install_path.split('\\').slice(-1)[0]
-    } else {
-      newInstallPath += '/' + install_path.split('/').slice(-1)[0]
-    }
+    newInstallPath = join(newInstallPath, basename(oldInstallPath))
+
+    const command = `mv -f '${oldInstallPath}' '${newInstallPath}'`
```

Comment:
Just a reminder to test this on windows because might not be working.

Reply:
I haven't changed the command, I just moved it to up here to display it in the log

Reply:
Yes, its a reminder for me tbh because I forgot to check if was working on windows and I believe someone said on discord that it was broken since we should be using the `Move-Item` instead, but that's another issue not related with this PR.

Reply:
`mv` is an alias for `Move-Item` on Windows. Although I don't know how well it handles that `-f` parameter, it should be `-force`